src/main/java/edu/cornell/kfs/coa/batch/CuCoaBatchConstants.java

```diff
@@ -21,4 +21,47 @@
         
         public static final String OUTPUT_FILE_NAME = "kfsClosedAccounts";
     }
+    
+    public static class WorkdayOpenAccountsFileCreationConstants {
+        public static final String OUTPUT_FILE_NAME = "kfsOpenAccountsSubaccountsSubobjectcodes";
+    }
+    
+    public enum WorkdayOpenAccountDetailDTOCsvColumn{
```

Comment:
Missing whitespace before the curly brace.

Reply:
ok